src/lib.rs

```diff
@@ -14,9 +14,13 @@ pub mod spend;
 mod testutils;
 
 pub use bip39;
+use bitcoin::electrum;
 pub use miniscript;
 
-pub use crate::bitcoin::d::{BitcoinD, BitcoindError, WalletError};
+pub use crate::bitcoin::{
+    d::{BitcoinD, BitcoindError, WalletError},
+    electrum::{Electrum, ElectrumError},
```

Comment:
It might be useful to also expose `ElectrumClient` so that the GUI can use it to check connectivity given the user's `ElectrumConfig` from the installer.